extensions/objects/templates/algebraic-expression-editor.component.html

```diff
@@ -8,7 +8,7 @@
 
 <style>
   .guppy-div-creator.guppy_active, .guppy-div-creator.guppy_inactive {
-    width: 550px;
+    width: 350px;
```

Comment:
Won't this reduce its width for normal screens? Is that intended?

Reply:
This is how it looks in the add response modal:
![Screenshot from 2020-08-27 23-17-02](https://user-images.githubusercontent.com/35144226/91477127-92582500-e8bb-11ea-9629-d128b5c776c8.png)

Since this is now consistent with the rule dropdown, it seems cleaner.
Also, note that the learner's UI is unaffected by this.

Reply:
Ah ok, lgtm!